api/app/tests/Unit/Entity/DeputyTest.php

```diff
@@ -1,39 +0,0 @@
-<?php
```

Comment:
This entire file was duplicated under the integration test. Removed this test file & renamed the other